aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
 Publishing to IIS
-=============================
+=================
 
-By `Rick Anderson`_ and `Luke Latham`_
+By `Rick Anderson`_ and `Luke Latham <https://github.com/GuardRex>`_
 
-.. contents:: Sections:
-  :local:
-  :depth: 1
+- `Supported Operating Systems`_
+- `IIS Configuration`_
+- `Install the Microsoft ASP.NET Core Module`_
+- `Install .NET Core on the Server`_
+- `Deploy the Application`_
+- `Configure the Website in IIS`_
+- `Create a Data Protection Registry Hive`_
+- `Common Errors`_
+- `Additional Resources`_
 
-Install the HTTP Platform Handler
------------------------------------
+Supported Operating Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Install the HTTP Platform Handler version 1.2 or higher:
+The following operating systems are supported:
 
-  - `64 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkID=690721>`_
-  - `32 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkId=690722>`_
+- Windows 7 and newer
+- Windows Server 2008 R2 and newer\*
 
-If you need to enable IIS, see `IIS server configuration`_.
+\*Conceptually, the IIS configuration described in this document also applies to hosting ASP.NET Core applications on Nano Server IIS, but refer to `ASP.NET Core on Nano Server <https://docs.asp.net/en/latest/tutorials/nano-server.html>`_ for specific instructions.
 
-Configure Data Protection
------------------------------------
+IIS Configuration
```

Comment:
The problem is that we don't want to maintain docs for technologies that we don't directly own. What happens if the IIS team makes some changes that affect these troubleshooting steps? These docs are the ASP.NET Core docs, not the IIS docs. That doesn't mean we can't link to the IIS docs liberally, but we need to make sure we document ASP.NET Core before we try to document other technologies. We have plenty that we still need to document for ASP.NET Core.